tests/test_cli.py

```diff
@@ -1,9 +1,28 @@
 import subprocess
 import sys
+import tempfile
+from pathlib import Path
 
+from conftest import run_cli
 from deploy_tools import __version__
 
+PATH_TO_SCHEMAS = (
+    Path(__file__).parent.parent / "src" / "deploy_tools" / "models" / "schemas"
+)
+
 
 def test_cli_version():
     cmd = [sys.executable, "-m", "deploy_tools", "--version"]
```

Comment:
Is it possible to use run_cli here, too?

Reply:
Perhaps, but I'm OK with this staying like the copier template provides it. That way we can pick up changes in the template if that ever happens.

Reply:
Yep, I have my doubts about CliRunner in general, and I think having any system testing at all is a major improvement over the current situation.

Once I've started to go through and add unit testing, we can look at other ways to organise this code.

Reply:
:+1: